src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(numb1) {
+    return function(numb2) {
+      counter++;
+
+      if (counter > 3 && counter % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
```

Comment:
I think, you can done your code without "else"

Reply:
agree